src/WakaTime.Shared.ExtensionUtils/Enums.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WakaTime.Shared.ExtensionUtils
+{
+    public enum HeartbeatCategory
+    {
+        [Description("coding")]
+        Coding,
+
+        [Description("building")]
+        Building,
+
+        [Description("indexing")]
+        Indexing,
+
+        [Description("debugging")]
+        Debugging,
+
+        [Description("running_tests")]
```

Comment:
The cli expects "running tests" (considering a space between word).

Reply:
That is true, I forgot that. I wrote the description as the enum was named before.